docs/api/js/accounts-l1-l2.md

```diff
@@ -39,13 +39,13 @@ async approveERC20(
 import * as zksync from "zksync-web3";
 import { ethers } from "ethers";
 
-const PRIVATE_KEY = "0xc8acb475bb76a4b8ee36ea4d0e516a755a17fad2e84427d5559b37b544d9ba5a";
+const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
 
 const zkSyncProvider = new zksync.Provider("https://testnet.era.zksync.dev/");
 const ethereumProvider = ethers.getDefaultProvider("goerli");
 const wallet = new zksync.Wallet(PRIVATE_KEY, zkSyncProvider, ethereumProvider);
```

Comment:
Somewhere I see that we keep 
```
import * as zksync from "zksync-web3";
new zksync.Provider(); or new zksync.Wallet();
```
structure.

Somewhere I see updated:
```
import { Wallet, Provider } from "zksync-web3";
new Provider(); or new Wallet();
```

I would suggest to use one standard in all examples.